pyslackers_website/templates/slack/partials/member_history_chart.html

```diff
@@ -0,0 +1,55 @@
+
+<script src="https://cdnjs.cloudflare.com/ajax/libs/Chart.js/2.7.1/Chart.min.js"></script>
+
+<div class="ui container segment">
+  <canvas id="members_history_chart" width="400" height="155">
+    Your browser cannot display this graph. Please upgrade to HTML5-compliant browser to view.
+  </canvas>
+</div>
+
+<script>
```

Comment:
Since we are using a build system to build/minify our code we don't include scripts right in the template (instead it should be made a component in the existing [`client/slack/slack.js`](https://github.com/pyslackers/website/blob/153150dbe351f9ef2349cd9c3f49575db90796ca/client/slack/slack.js) and triggered on the page being ready (currently done via jquery: `$(() => { console.log('this code will not run until the page is ready!') })`)

Note: please don't follow the example of how I am passing the data over, that was meant as a short-term stop gap and it will be fixed shortly.

Reply:
Okay. Will give it another go tomorrow.
Thanks for the feedback.